armi/reactor/converters/uniformMesh.py

```diff
@@ -131,11 +131,12 @@ def convert(self, r=None):
                 f"the core's reference assembly mesh: {r.core.refAssem.getAxialMesh()}"
             )
             self.convReactor = self._sourceReactor
+            self.convReactor.core.updateAxialMesh()
             self._setParamsToUpdate()
             for assem in self.convReactor.core.getAssemblies(self._nonUniformMeshFlags):
                 homogAssem = self.makeAssemWithUniformMesh(
                     assem,
-                    self.convReactor.core.refAssem.getAxialMesh(),
+                    self.convReactor.core.p.axialMesh[1:],
```

Comment:
Has the `core.p.axialMesh` included in parameters been thermally expanded?  (@albeanth My question is is if this axial mesh on the parameters is always correct at this point in time. I can't tell.)

Reply:
Since we call `self.convReactor.core.updateAxialMesh()` immediately above, the `p.axialMesh` will be consistent with the `refAssem` axial mesh, which should be thermally expanded at this point.

https://github.com/terrapower/armi/blob/8757794cac263175430989da31bb504e345bdd0d/armi/reactor/reactors.py#L1830-L1866